From e821ef9acc4fd43200d5f3a6cf2365186920511d Mon Sep 17 00:00:00 2001
From: Chen Meng J <meng.j.chen@intel.com>
Date: Mon, 3 Dec 2018 16:14:33 +0800
Subject: [PATCH 0993/1214] media: i2c: add required kconf dependency

add required kconf dependency for:
ti960, ti964, max9286.

Change-Id: Ifed1a0407e7cbdaa9d6251681f7b5b9e997c1edc
Tracked-On: PKT-1732
Signed-off-by: Chen Meng J <meng.j.chen@intel.com>
---
 drivers/media/i2c/Kconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/media/i2c/Kconfig b/drivers/media/i2c/Kconfig
index 07f112a..e17d9fe 100644
--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -1072,19 +1072,19 @@ config VIDEO_I2C
 
 config VIDEO_TI964
 	tristate "TI964 driver support"
-	depends on I2C && VIDEO_V4L2
+	depends on I2C && MEDIA_CONTROLLER && VIDEO_V4L2 && VIDEO_V4L2_SUBDEV_API
 	---help---
 	This is a driver for TI964 camera.
 
 config VIDEO_MAX9286
 	tristate "MAX96705/MAX9286 Serializer/Deserializer"
-	depends on I2C && VIDEO_V4L2 && REGMAP_I2C
+	depends on I2C && MEDIA_CONTROLLER && VIDEO_V4L2 && VIDEO_V4L2_SUBDEV_API && REGMAP_I2C
 	---help---
 	This is a MAXIM 96705 Serializer and MAXIM 9286 CSI-2 Deserializer driver.
 
 config VIDEO_TI960
 	tristate "TI960 driver support"
-	depends on I2C && VIDEO_V4L2
+	depends on I2C && MEDIA_CONTROLLER && VIDEO_V4L2 && VIDEO_V4L2_SUBDEV_API
 	---help---
 	This is a driver for TI960 Deserializer.
 
-- 
2.7.4

